backend/chatbot/chat_router.py

```
@@ -0,0 +1,224 @@
+RABBITMQ_URL = os.getenv("RABBITMQ_URL", "amqp://guest:guest@localhost/")
+
+# Connection pool for RabbitMQ
+rabbitmq_connection_pool = None
+
+async def get_rabbitmq_connection() -> aio_pika.RobustConnection:
+    global rabbitmq_connection_pool
+    if not rabbitmq_connection_pool:
+        rabbitmq_connection_pool = await aio_pika.connect_robust(RABBITMQ_URL)
+    return rabbitmq_connection_pool
+
+async def verify_token(token: HTTPAuthorizationCredentials = Depends(security)) -> dict:
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)